pulse card rendering fails on question with decimal values when using sql

not sure if this is specific to SQL usage or not, but this error is being generated when trying to add a card to a pulse when the card has decimal values in the data.

```metabase.pulse :: Pulse card render error:java.lang.ArithmeticException: Non-terminating decimal expansion; no exact representable decimal result.```

Sadly, after applying this fix to some running MB instances I'm still seeing an error ...

```
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:  2016-01-28 22:25:59,620 [ERROR] metabase.middleware :: GET /api/pulse/preview_card_info/928 500 (1 s)
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:  {:message
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:   "Non-terminating decimal expansion; no exact representable decimal result.",
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:   :stacktrace
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:   ["metabase.pulse$render_table$fn__14585.invoke(pulse.clj:245)"
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:    "metabase.api.pulse$fn__22616$fn__22617.invoke(pulse.clj:99)"
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:    "metabase.api.common.internal$wrap_catch_api_exceptions.invoke(internal.clj:216)"
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:    "metabase.api.pulse$fn__22616.invoke(pulse.clj:91)"
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:    "metabase.middleware$enforce_authentication$fn__23234.invoke(middleware.clj:82)"
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:    "metabase.api.routes$fn__23354.invoke(routes.clj:49)"
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:    "metabase.routes$fn__23910.invoke(routes.clj:23)"
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:    "metabase.middleware$log_api_call$fn__23327.invoke(middleware.clj:256)"
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:    "metabase.middleware$add_security_headers$fn__23287.invoke(middleware.clj:193)"
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:    "metabase.middleware$bind_current_user$fn__23238.invoke(middleware.clj:95)"
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:    "metabase.middleware$wrap_current_user_id$fn__23212.invoke(middleware.clj:64)"
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:    "metabase.middleware$wrap_api_key$fn__23253.invoke(middleware.clj:106)"
Jan 28 14:25:59 metabase-spot 8c88df003ac6-stdouterr.log:    "metabase.middleware$wrap_session_id$fn__23205.invoke(middleware.clj:55)"]}
```

--- a/src/metabase/pulse.clj
+++ b/src/metabase/pulse.clj
@@ -242,7 +242,7 @@
                                                :max-height       :10px
                                                :height           :10px
                                                :border-radius    :2px
-                                               :width            (str (float (* 100 (/ (bar-column row) max-value))) "%")})}
+                                               :width            (str (float (* 100 (/ (double (bar-column row)) max-value))) "%")})} ; cast to double to avoid "Non-terminating decimal expansion" errors
                           "&#160;"]])])
                    rows)]]))
 

